core/runtime/runtime_api/impl/parachain_host.cpp

```diff
@@ -213,6 +225,7 @@ namespace kagome::runtime {
     availability_cores_.erase(blocks);
     session_index_for_child_.erase(blocks);
     candidate_pending_availability_.erase(blocks);
+    candidates_pending_availability_.erase(blocks);
```

Comment:
The similarity of these two names looks pretty error-prone to me, can the name be changed to be more distinct?

Reply:
candidates_pending_availability_ is the whole point of issue. Both names are used in polkadot-sdk as well